stdlib/build.rs

```diff
@@ -25,51 +23,23 @@ fn main() -> Result<()> {
     println!("cargo:rerun-if-changed=asm");
     println!("cargo:rerun-if-changed=../assembly/src");
 
+    let manifest_dir = env!("CARGO_MANIFEST_DIR");
+    let asm_dir = Path::new(manifest_dir).join(ASM_DIR_PATH);
+
     let namespace = "std".parse::<LibraryNamespace>().expect("invalid base namespace");
-    let version = env!("CARGO_PKG_VERSION").parse::<Version>().expect("invalid cargo version");
-    let stdlib = MaslLibrary::read_from_dir(ASM_DIR_PATH, namespace, version)?;
+    // TODO: Add version to `Library`
+    let _version = env!("CARGO_PKG_VERSION").parse::<Version>().expect("invalid cargo version");
```

Comment:
We should probably do this before the release.

Reply:
I think we should probably define the semantics of library versioning first, since it isn't super clear to me just yet what one can do with a version that makes sense at the moment. The only thing useful at this point, is the `digest` computed for a `CompiledLibrary`, which tells you when the contents of the library has changed.